BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -406,7 +406,13 @@ extension ScrollView
         // Do the same adjustment to our expected content inset.
         
         if case .refreshing = refreshControlState {
```

Comment:
Minor, it might be nice to put this line and the comments above all within the `else`